packages/gatsby-theme-patternfly-org/components/example/example.js

```diff
@@ -56,7 +57,10 @@ export const Example = props => {
   }
   const exampleName = title.replace(/-/g, ' ').replace(/  /g, '-');
   const fullscreenLink = `${location.pathname}/${slugger(title.toLowerCase())}`;
-  const scope = useMDXScope();
+  const scope = {
+    ...PatternflyReactCore,
```

Comment:
This is so React devs don't have to `import { Stuff } from '@patternfly/react-core';` at the top of each MDX file. The whole thing is included in the bundle anyways so we might as well inject it in each example.